packages/forms/src/Components/Select.php

```diff
@@ -42,6 +42,8 @@ class Select extends Field implements Contracts\HasNestedRecursiveValidationRule
 
     protected ?Closure $createOptionUsing = null;
 
+    protected string | Closure | null $createOptionFormLabelUsing = null;
```

Comment:
Shouldn't this be something like `createOptionModalHeading`?

Reply:
yes it could be, do you want that i change ?

Reply:
The `Using` prefix isn't required, right? Since you could also just pass a string.

Reply:
That's right, I tried to copy how was structured the other field